monitoring/shared.go

```diff
@@ -107,6 +111,17 @@ var sharedContainerCPUUsage sharedObservable = func(containerName string) Observ
 	}
 }
 
+var sharedContainerFsInodes sharedObservable = func(containerName string) Observable {
+	return Observable{
+		Name:              "fs_inodes_used",
+		Description:       "fs inodes in use by instance",
+		Query:             fmt.Sprintf(`sum by (instance) (container_fs_inodes_total{%s})`, promCadvisorContainerMatchers(containerName)),
```

Comment:
```suggestion
		Query:             fmt.Sprintf(`sum by (instance)(container_fs_inodes_total{%s})`, promCadvisorContainerMatchers(containerName)),
```

For consistency with how we write it elsewhere